l2geth/internal/ethapi/api.go

```diff
@@ -798,10 +798,13 @@ func DoCall(ctx context.Context, b Backend, args CallArgs, blockNrOrHash rpc.Blo
 	}
 	// Set sender address or use a default if none specified
 	var addr common.Address
+	// Set sender address or use a default if none specified
```

Comment:
I think this comment may have been duplicated twice in error

Reply:
Otherwise looking really good. I think it could be worth adding an integration test that demonstrates that the address used by default for `eth_call` is address(0) unless the `from` address is provided.

Reply:
The integration test could involve deploying a contract with a function that returns `msg.sender` and then using `eth_call` to call the function with and without setting from in the `eth_call`

Reply:
+2 tests 👍 

Let me know if everything is good. :) 
Do you prefer if I squash the 2 commits into 1 ? 

ps: I've also included a linting fix